geth-utils/gethutil/mpt/witness/gen_witness_transactions_test.go

```diff
@@ -16,11 +16,54 @@ import (
 )
 
 func TestTransactions(t *testing.T) {
```

Comment:
We can rename this test for clarity.
```suggestion
func TestTransactionStackTrie(t *testing.T) {
```

Reply:
I renamed it to `TestNonHashedTransactionsStackTrieGetProof` as it is about getting the proof from the stack trie where the nodes are not hashed.